open_event/views/views.py

```diff
@@ -278,6 +281,38 @@ def generate_icalender_track(track_id):
 	cal.add_component(track)
 	return cal.to_ical()
 
+@app.route('/api/v1/session/<int:session_id>/reviews', methods=['GET'])
```

Comment:
The POST method will handle updating existing review.. Should I use PUT for update?